### ISSUE TITLE
Does not stylize the custom elements that are not direct children of the ShadowRoot

### ISSUE THREAD

**Issue by Lodin** [calebdwilliams/construct-style-sheets#7]
I'm continue experimenting with the polyfill and found that it does not stylize element if it is not a direct child of the `ShadowRoot`. E.g., like the following:
```html
<x-foo>
  #shadow-root
    <x-bar></x-bar> // stylized
    <div>
        <x-baz></x-baz> // not stylized
    </div>
</x-foo>
```

I found the source of the problem and going to create a PR to solve it.

**Comment by calebdwilliams** [comment]
Could you rephrase the issue? I’m. It sure I follow. What is the current behavior and what is the expected behavior? Could you provide a reproduction here?

**Comment by Lodin** [comment]
@calebdwilliams, yeah, sure, sorry for that. I was too hasty.
I added a repro and a brief description. BTW, creating a repro, I found another bug I didn't fix in my PR 😩.
